php-fpm_exporter.yaml

```diff
@@ -34,7 +34,7 @@ pipeline:
 
   - uses: goreleaser/build
     with:
-      args: --skip-validate
+      skip: "docker,ok,publish,validate"
```

Comment:
this is blocked by https://github.com/chainguard-dev/melange/pull/1344/files#diff-13bad82f626ca11a5d89c0964414b99d5b70cb5c389754e2ece24d240467b62bR14